src/main/java/de/baleipzig/iris/ui/presenter/neuralnetconfig/NeuralNetConfigPresenter.java

```diff
@@ -0,0 +1,61 @@
+package de.baleipzig.iris.ui.presenter.neuralnetconfig;
+
+import com.vaadin.ui.UI;
+import de.baleipzig.iris.model.neuralnet.neuralnet.NeuralNetMetaData;
+import de.baleipzig.iris.ui.presenter.base.BaseSearchNNPresenter;
+import de.baleipzig.iris.ui.service.neuralnetconfig.INeuralNetConfigService;
+import de.baleipzig.iris.ui.view.neuralnetconfig.INeuralNetConfigView;
+import de.baleipzig.iris.ui.view.training.TrainingView;
+import de.baleipzig.iris.ui.viewmodel.neuralnetconfig.NeuralNetConfigViewModel;
+
+
+public class NeuralNetConfigPresenter extends BaseSearchNNPresenter<INeuralNetConfigView, INeuralNetConfigService> {
+
+    private NeuralNetConfigViewModel model = new NeuralNetConfigViewModel();
+
+    public NeuralNetConfigPresenter(INeuralNetConfigView view, INeuralNetConfigService service) {
+
+        super(view, service);
+    }
+
+    @Override
+    public void init() {
+
+        super.init();
+        initViewModel();
+        bindViewModelToView();
+    }
+
+    @Override
+    public void handleSelection(NeuralNetMetaData metaData) {
+
+        model.setSelectedNeuralNetId(metaData.getId());
+        model.setNeuralNet(service.getNeuralNetWorker().load(metaData.getId()));
+        model.setDescription(metaData.getDescription());
+        model.setName(metaData.getName());
+        model.setNetStructure(service.getNeuralNetWorker().toJson(model.getNeuralNet()));
+
+        UI.getCurrent().access(() -> view.update(model));
+    }
+
+    public  Void redirectToTrainingView() {
+
+        String parameterString = model.getSelectedNeuralNetId() == null ? "" : String.format("/%s=%s", "uuid", model.getSelectedNeuralNetId());
```

Comment:
kannst du die url generierung in den helper verschieben?
               String url = urlhelper.getUrl(String viewName, Map<String, String> param)
dann ist alles an einem platz und und man braucht sich um die parameter-codierung nicht mehr kümmern

UI.getCurrent().getNavigator().navigateTo(url)
diesen teil würde ich als protetected methode in die baseview legen        
entweder als navigateTo(url) oder als navigateTo(viewname, parametermap). finde variante 2 schöner